### PR TITLE
Add Aju

### PR DESCRIPTION
**Pull request by Aju100**
#### The main changes are

-  Added new developer 
-  Added ignore files of go lang and vs code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
```
Comment (ujwaldhakal) [inline .gitignore]: i think these files are related with c languages so its better to use global gitignore  so only go related git ignore will be placed :) Please make necessary adjustments @Aju100

Comment (Aju100) [reply .gitignore]: sure

Comment (Aju100) [reply .gitignore]: @ujwaldhakal  can u review code.

Comment (ujwaldhakal) [reply .gitignore]: @Aju100 can you rebase with master and squash 3 commits into 1 so i can merge it

Comment (Aju100) [reply .gitignore]: ujwaldhakal i am not able to rebase and squash 3 commits into 1 . But everything is right, isn't it?

Comment (ujwaldhakal) [reply .gitignore]: yes just it makes more commit and hard for review ok let me do it for you

Comment (ujwaldhakal) [reply .gitignore]: git pull --rebase origin master & git rebase -i HEAD~${numberofcommits}

Comment (ujwaldhakal) [reply .gitignore]: git pull Aju100 master && git rebase -i HEAD~4

Comment (Aju100) [reply .gitignore]: ok ujwaldhakal I will do

**Comment by ujwaldhakal** [comment]
added

**Pull request closed without merging**